components/automate-cli/pkg/verifyserver/services/batchcheckservice/trigger/hardwareresourcecountchecktrigger.go

```diff
@@ -1,15 +1,91 @@
 package trigger
 
```

Comment:
File name should be readable to the user, add "_"

Reply:
Not needed. Please keep like this only